src/vorta/notifications.py

```diff
@@ -20,25 +25,44 @@ def pick(cls):
 
 
 class DarwinNotifications(VortaNotifications):
-    def deliver(self, title, text):
-        if SettingsModel.get(key='enable_notifications').value:
-            from Foundation import NSUserNotification
-            from Foundation import NSUserNotificationCenter
 
-            notification = NSUserNotification.alloc().init()
-            notification.setTitle_(title)
-            notification.setInformativeText_(text)
-            center = NSUserNotificationCenter.defaultUserNotificationCenter()
-            if center is not None:  # Only works when run from app bundle.
-                center.deliverNotification_(notification)
+    def deliver(self, title, text, level='info'):
+        if not SettingsModel.get(key='enable_notifications').value:
```

Comment:
Some duplicate code here that decides on whether to send a notification or not. Moved this to a single function.